pkg/apis/core/v1alpha1/types.go

```diff
@@ -67,66 +74,69 @@ type DataVolumeSource struct {
 
 // DataVolumeSourcePVC provides the parameters to create a Data Volume from an existing PVC
 type DataVolumeSourcePVC struct {
-	Namespace string `json:"namespace,omitempty"`
-	Name      string `json:"name,omitempty"`
+	// The namespace of the source PVC
+	Namespace string `json:"namespace"`
```

Comment:
Yes, the name and namespace are required, so omitempty should not have been there. Now we had it correctly in our code schema (that they are required) so technically not a bug. But I want to prevent drift between the two so I have to make sure the generated from types.go and the generated from our code schema are the same.